README.md

```diff
@@ -61,7 +69,7 @@ FederatedScope
 └── setup.py
 ```
 
-## Quick Start
+<details><summary><h2> Quick Start </h2></summary>
```

Comment:
Please change it to `## Quick Start` Since we do not want to fold the quick start section

Reply:
Won't doing this will make the readme look lengthy ?

Reply:
We believe that these sections are very important for users to get started and do want to fold them, thx!